_announcements/01_apple_common_removal.md

```diff
@@ -0,0 +1,319 @@
+---
+
+Upcoming changes regarding Apple's Root Certificate Program could impact your agency. The Federal PKI Policy Authority (FPKIPA) has elected to remove our U.S. Government Root CA certificate (Federal Common Policy CA [FCPCA/COMMON]) from Apple's pre-installed Operating System Trust Stores. 
+
+{% include alert-info.html content="This announcement will be updated as more information and additional procedures become available. Please watch for additional updates from the Federal PKI listservs, ICAM listservs, and the ICAM Subcommittee." %}
+
+Starting in **September-October 2018**, government users of Apple iOS, macOS, and tvOS devices will receive errors when encountering instances of a Federal PKI CA-issued certificate. You can mitigate the impact for government intranets and government-furnished equipment by using configuration management tools for federal devices.
+
+{% include alert-info.html content="The FPKIPA has also elected to remove the Federal Common Policy CA root certificate from Microsoft's Certificate Trust List." %}
+
+- [How Does this Work?](#how-does-this-work)
+- [What Will Be Impacted?](#what-will-be-impacted)
+- [What Should I Do?](#what-should-i-do)
+- [Frequently Asked Questions](#frequently-asked-questions)
+- [Additional Resources](#additional-resources)
+
+## How Does This Work?
+
+Apple currently distributes the Federal Common Policy CA (FCPCA/COMMON) through its pre-installed Operating System Trust Stores for iOS, macOS, and tvOS. 
+
+Three root CA certificate _types_ reside in Apple's Trust Stores:
+
+- _Trusted Certificates_ &mdash; Trusted certificates that establish a chain of trust.
+- _Always Ask_ &mdash; Untrusted certificates that are not blocked. If a resource (e.g., website or signed email) chains to one of these certificates, the Apple Operating System will ask you to choose whether or not to trust it.
+- _Blocked_ &mdash; Potentially compromised certificates that will never be trusted.
+
+These certificate types are stored within Apple _Keychains_:
+
+- _Login Keychain_ &mdash; Certificates associated with a user account logged into a device.
+- _System Keychain_ &mdash; Certificates associated with all user accounts on a device (similar to the Microsoft Windows' _Local Machine_ certificate store). 
+- _System Roots Keychain_ &mdash; Includes Apple's _pre-installed_, trusted root CA certificates. COMMON will be removed from this Keychain.
+ 
+## What Will Be Impacted?
+
+These Apple Operating System versions will be impacted:
+
+|**macOS**|**iOS**|**tvOS**|
+| :-------- |:-------- |:-------- |
+| High Sierra (10.13) |	 iOS11 | tvOS11 |
+| Sierra (10.12) | iOS10 | tvOS10 |
+| El Capitan (10.11) | iOS9 | |
```

Comment:
I don't think this is true?  it won't be retroactively applied to 10.11, 10.10 or 10.9 - or iOS7, iOS8, iOS9, and iOS10?

Check the apple notes - it's forward applied?

Reply:
You're right - the note says "Removals are usually only forward - Removals can be backwards when there is a security concern via a revocation mechanism".

I mis-remembered that, or possibly confused it with the Symantec resolution, thanks for catching! I'll update to remove the older OS versions.